components/Home/index.tsx

```diff
@@ -310,7 +309,7 @@ export default function Home({ metricsData, siteConfigData }: HomeProps) {
                 variant={ButtonVariant.LIGHT}
               />
               <Button
-                href={'https://calendly.com/d/zxg-m2m-54p'}
+                href={'mailto:bd@cow.fi'}
```

Comment:
i should have said in the PR, and not too important. But we have these things in constants. It would be best to use it